### PR TITLE
feat: doc for audiomonophoner crafter

### PR DESCRIPTION
**Pull request by rutujasurve94**
Doc for `audiomonophoner` crafter

### CODE REVIEW

File: crafters/audio/AudioMonophoner/README.md [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
-makes the audio signal monophonic on doc-level. 
\ No newline at end of file
+**AudioMonophoner** makes the audio signal monophonic on doc-level.
+
+The supported sound formats are:
+WAV, MP3, OGG, AU,FLAC, RAW, AIFF, AIFF-C, PAF, SVX, NIST, VOC, IRCAM, W64, MAT4, MAT5, PVF, XI, HTK, SDS, AVR, WAVEX, SD2, CAF, WVE, MPC2K, RF64.
+
+## Usage
+
+The **AudioMonophoner** executor uses the `librosa` library to convert the input `blob` (`ndarray` representation of the audio signal) to a dictionary of `monophonic` audio signal.
```
Comment (nan-wang) [inline crafters/audio/AudioMonophoner/README.md:10]:
```suggestion
The **AudioMonophoner** executor uses the [`librosa`](https://github.com/librosa/librosa) library to convert the input `blob` (`ndarray` representation of the audio signal) to a dictionary of `monophonic` audio signal. 
```

File: crafters/audio/AudioMonophoner/README.md [hunk 2]
```diff
@@ -1,3 +1,43 @@
...
+## Usage
+
+The **AudioMonophoner** executor uses the `librosa` library to convert the input `blob` (`ndarray` representation of the audio signal) to a dictionary of `monophonic` audio signal. 
+
+We use Pod images in several ways:
+
+1. Run with Docker: `docker run`
+   ```bash
+    docker run jinahub/pod.crafter.audiomonophoner:0.0.9-1.0.1 --port-in 55555 --port-out 55556
+    ```
```
Comment (nan-wang) [inline crafters/audio/AudioMonophoner/README.md:17]: The pod won't be usable.
```suggestion
1. Run with Docker: `docker run`
   ```bash
    docker run jinahub/pod.crafter.audiomonophoner:0.0.9-1.0.1 --port-in 55555 --port-out 55556
    ```
```

File: crafters/audio/AudioMonophoner/README.md [hunk 3]
```diff
@@ -1,3 +1,43 @@
...
+    jina pod --uses hub/example/audiomonophoner.yml --port-in 55555 --port-out 55556
+   ```
+    
+5. Docker command
+
+   Specify the image name along with the version tag. In this example we use Jina version `1.0.1`
+
+   ```bash
+    docker pull jinahub/pod.crafter.audiomonophoner:0.0.9-1.0.1
+    ```
```
Comment (nan-wang) [inline crafters/audio/AudioMonophoner/README.md:43]:
```suggestion

```

**Review by nan-wang — CHANGES_REQUESTED** [review]

File: crafters/audio/AudioMonophoner/README.md (hunk 2 above)
Comment (rutujasurve94) [reply crafters/audio/AudioMonophoner/README.md]: Binding the container port here

**Review by bwanglzu — DISMISSED** [review]
LGTM!

**Review by Yongxuanzhang — APPROVED** [review]
LGTM! 👍  well done!

File: crafters/audio/AudioMonophoner/manifest.yml [hunk 4]
```diff
@@ -7,6 +7,6 @@ author: Jina AI Dev-Team (dev-team@jina.ai)
 url: https://jina.ai
 vendor: Jina AI Limited
 documentation: https://github.com/jina-ai/jina-hub
```
Comment (deepankarm) [inline crafters/audio/AudioMonophoner/manifest.yml:9]: We need to point to the right readme for every executor.

File: crafters/audio/AudioMonophoner/README.md [hunk 5]
```diff
@@ -1,3 +1,35 @@
...
+The supported sound formats are:
+WAV, MP3, OGG, AU,FLAC, RAW, AIFF, AIFF-C, PAF, SVX, NIST, VOC, IRCAM, W64, MAT4, MAT5, PVF, XI, HTK, SDS, AVR, WAVEX, SD2, CAF, WVE, MPC2K, RF64.
+
+## Usage
+
+The **AudioMonophoner** executor uses the [`librosa`](https://github.com/librosa/librosa)  library to convert the input `blob` (`ndarray` representation of the audio signal) to a dictionary of `monophonic` audio signal. 
+
+We use Pod images in several ways:
+
+1. Run with Docker: `docker run`
```
Comment (deepankarm) [inline crafters/audio/AudioMonophoner/README.md:14]: Why do we promote `docker run` first? It should be the last option.

File: crafters/audio/AudioMonophoner/README.md [hunk 6]
```diff
@@ -1,3 +1,35 @@
...
+    ```
+    
+3. Run with Jina CLI
+   ```bash
+    jina pod --uses docker://jinahub/pod.crafter.audiomonophoner:0.0.9-1.0.1 --port-in 55555 --port-out 55556
+    ```
+    
+4. Run with conventional local usage. Needs `uses` argument:
+   ```bash
+    jina pod --uses hub/example/audiomonophoner.yml --port-in 55555 --port-out 55556
```
Comment (deepankarm) [inline crafters/audio/AudioMonophoner/README.md:34]: `hub/example/audiomonophoner.yml` - Let's change it to the correct relative path - `crafters/audio/AudioMonophoner/audiomonophoner.yml`

**Review by deepankarm — CHANGES_REQUESTED** [review]

**Comment by rutujasurve94** [comment]
@deepankarm thanks for the review, I've incorporated the changes in this PR

**Review by deepankarm — APPROVED** [review]

**Pull request merged**